Assets/_BForBoss/_Character/Scripts/PlayerDashBehaviour.cs

```diff
@@ -12,7 +12,8 @@ namespace Perigon.Character
     public class PlayerDashBehaviour : MonoBehaviour
     {
         [Title("Visual Effects")]
-        [SerializeField] private ParticleSystem _dashVisualEffects = null;
+        [SerializeField] private GameObject _dashEffect = null;
```

Comment:
So because you have `private DashLinesEffectBehaviour _dashEffectBehaviour;` below, you can make that a `SerializeField` instead, and pass it in there :) 

And then under `Awake`, check 
```
 if (_dashEffect == null)
            {
                Debug.LogWarning("Dash Effect Missing from PlayerDashBehaviour.cs");
            }
```

Reply:
Would it make more sense to put the error check under `Awake()` or `SetupVisualEffects()`?

Reply:
I definitely didn't realize you could directly pass a GameObject's Script Component instances through Serialize Fields! That is a much clearer solution, thanks for letting me know! :)

Reply:
Oh I think `Setup Visual Effects`  would be nice.